extlinks/settings/base.py

```diff
@@ -144,6 +144,7 @@
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 
 # Cron config
+DJANGO_CRON_LOCK_BACKEND = "django_cron.backends.lock.database.DatabaseLock"
```

Comment:
This is to ensure the locking is done via db rather than cache correct?

Reply:
Yeah, the cache-based lock provider uses a simple time-based lock that uses the same timeout value across all jobs:
https://github.com/Tivix/django-cron/blob/master/django_cron/backends/lock/cache.py#L21